TwitchDownloaderCore/TwitchObjects/ChatRoot.cs

```diff
@@ -208,7 +208,21 @@ public class EmbedEmoteData
         public int height { get; set; }
     }
 
+    public class EmbedChatBadgeData
+    {
+        public string title { get; set; }
+        public string description { get; set; }
+        public string url { get; set; }
+    }
```

Comment:
Nitpick: An empty newline between class declarations to keep formatting consistent would be nice

Reply:
Fixed. 
_I missed it an another place and fixed there as well._